### ISSUE TITLE
Update Skip to Section Menu Display

### ISSUE THREAD

**Issue by kevinreiss** [pulibrary/allsearch_frontend#274]
### Current display
![skip to section](https://github.com/pulibrary/allsearch_frontend/assets/9905193/42539dd7-1573-46fc-94d5-cb9a7232cffc)

### Desired display
Skip to section menu options:
- [x] Remove underline text
- [x] Centered text
- [x] On hover highlight the entire button 
- [x] Color contrast with the [current dark orange colors](https://github.com/pulibrary/allsearch_frontend/blob/039ac4a818b479ca9a4c85f1422028ffa75b0b16/assets/app.css#L179-L182) has a contrast of 3.02:
`Element has insufficient color contrast of 3.02 (foreground color: #ffffff, background color: #e77500, font size: 13.5pt (18px), font weight: normal). Expected contrast ratio of 4.5:1  `
- [x] the whole rectangle is clickable, not just the text

### Acceptance criteria

- [ ] I can successfully use this feature with Voice Over in Safari.
- [ ] I have checked this feature using one of the following browser extensions and confirmed that it does not add any level A or level AA issues:
  - axe dev tools
  - Lighthouse
  - Wave

## Open Questions
- What color should we use for the highlighting mouseover of the issue, we have a color contrast issue with the recommendations from WDS. 

- If we change the skip to section link color to #212529 (it is gray-90 according to the WDS styleguide from OIT) and we use the dark orange  from OIT  which is --orange-50: #E77500
The skip to section will look like that when you hover over one of the links
It passes the color contrast for normal text. It only fails for WCAG AAA which is not official yet.
![Screenshot 2024-04-23 at 12 58 48 PM](https://github.com/pulibrary/allsearch_frontend/assets/9905193/6b27dd1e-7e05-4476-a772-810a2d69795b)

**Comment by rladdusaw** [comment]
The orange 20 color and black text have a contrast ratio of about 17:1

<img width="503" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBN2lNQkE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--35d2dcac234fec752b09b6bd5815815963ccaea4/Screenshot%202024-04-24%20at%209.45.02%E2%80%AFAM.png" alt="Screenshot 2024-04-24 at 9.45.02 AM.png" />
